CHANGES.txt

```diff
@@ -15,7 +15,18 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
-- Renamed ec2 ``cloud.aws.*`` settings to ``discovery.ec2.*`` (eg. 
+- When using S3 for ``COPY FROM`` and ``COPY TO``, the credentials to access S3
```

Comment:
What's the reason for this change?

Reply:
Since we introduced the crate.keystore and the user can no longer provide credentials for S3 `CREATE REPOSITORY` through the SQL command (only through the keystore) we decided to do the same for `COPY FROM/TO` to be consistent regarding our security.